dashboards/cluster.jsonnet

```diff
@@ -125,12 +129,13 @@ local clusterCPUCommitment = graphPanel.new(
 
 
 local nodeCPUCommit = graphPanel.new(
-  'CPU commit %',
+  'Node CPU Commit %',
   formatY1='percentunit',
   description=|||
     % of each node guaranteed to pods on it
   |||,
   min=0,
+  max=1,
```

Comment:
Percentages can go over 100 and might represent error conditions. For example, if this query returns >1, I think it means autoscaling needs to kick in but hasn't.

Reply:
The commit % can't go over 1, can it? The scheduler won't assign pods to a node if that happens.

Reply:
I think it can happen with Evictions and when you have multiple schedulers running - that's when you get pods into 'OutOfMemory' state or similar, I think?